engines/bops_api/app/controllers/bops_api/application_controller.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
 module BopsApi
-  class ApplicationController < ActionController::Base
+  class ApplicationController < ActionController::API
+    include ActionController::MimeResponds
+    include BopsApi::ErrorHandler
```

Comment:
No need for the `BopsApi` prefix - Zeitwerk will look in the proper place first for `ErrorHandler`